server/events/delete_lock_command.go

```diff
@@ -0,0 +1,84 @@
+// Copyright 2017 HootSuite Media Inc.
```

Comment:
Don't need this boilerplate because this is a new file. That's only for old files.

Reply:
Removed